source/configuration_manual/mail_location/obox/obox_settings.rst

```diff
@@ -14,150 +14,243 @@ Enable obox plugin.
+             non-INBOX, non-specialuse folders and subsequently was moved to
+             another backend without flushing the indexes, Dovecot may not
+             realize that the user is now below quota. There are plans to
+             change this so that if user appears to be over quota, the quota
+             is strictly calculated before returning over quota failures or
+             executing any quota warning/over scripts.
 
 .. code-block:: none
 
-   metacache_priority_weights = 10% +1d 10% +1d 50% +1h 100% 0
-   metacache_size_weights = 2M +30 1G +120
+   plugin {
+     metacache_priority_weights = 10% +1d 10% +1d 50% +1h 100% 0
+     metacache_size_weights = 2M +30 1G +120
+   }
+
+Whenever metacache notices that ``metacache_max_space`` has been reached, it
+needs to delete some older index files to make space for new ones. This is
+done by calculating cleanup weights.
+
+The simplest cleanup weight is to just use the user's last access UNIX
+timestamp as the weight. The lowest weight gets deleted first.
 
-Whenever metacache notices that metacache_max_space has been reached, it needs to delete some older index files to make space for new ones. This is done by calculating cleanup weights. The simplest cleanup weight is to just use the user's last access UNIX timestamp as the weight. The lowest weight gets deleted first. It's possible to enable using only simple weights by explicitly setting ``metacache_priority_weights`` and ``metacache_size_weights`` to empty values. However, by default priorities are taken into account when calculating the weight:
+It's possible to enable using only simple weights by explicitly setting
+``metacache_priority_weights`` and ``metacache_size_weights`` to empty
+values. However, by default priorities are taken into account when calculating
+the weight.
 
-The metacache_priority_weights setting can be used to fine tune how metacache adjusts the cleanup weights for different index priorities. There are 4 major priorities (these are also visible in e.g. "doveadm metacache list" output):
+The ``metacache_priority_weights`` setting can be used to fine tune how
+metacache adjusts the cleanup weights for different index priorities. There
+are 4 major priorities (these are also visible in e.g. ``doveadm metacache
+list`` output):
 
  * 0 = User root indexes (highest priority)
  * 1 = FTS indexes
  * 2 = INBOX and \Junk folder indexes
- * 3 = other folders' indexes (lowest priority)
-
-The metacache_size_weights contains <percentage> <weight adjustment> pairs for each of these priorities.So for example the first "10% +1d" applies to the user root priority and the last "100% 0" applies to other folders' priority. The weight calculation is then done by:
-
- * Initial weight is the user's last access UNIX timestamp
- * metacache_priority_weights is next looked up for the given priority indexes, 
- 
- Example:
-
-.. code-block:: none 
-   
-    10% +1d
+ * 3 = Other folders' indexes (lowest priority)
```

Comment:
What does the `'` mean here?

Reply:
genitive of plural folder. Same as "Indexes of other folders". But .. this is a bit inconsistent because the other priorities don't use genitive either. I guess need to fix in some way.

Reply:
I made more clear that "INBOX" and "\Junk" are 'special' folders - so priority 3 is for "non-special folders"